<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>perf_graph</class>
 <widget class="QDialog" name="perf_graph">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1157</width>
    <height>900</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>AMD MIVisionX Inference Performance</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(235, 235, 235);</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="logo_label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/images/AMD.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="logo_label_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/images/Radeon.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="logo_label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/images/MIVisionX.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QGridLayout" name="gridLayout">
       <item row="16" column="0">
        <widget class="QLabel" name="ElapsedTimeName_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Elapsed Time:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLCDNumber" name="GPU_lcdNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="12" column="0">
        <widget class="QLabel" name="fps_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Images/Sec:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="15" column="0">
        <widget class="QLabel" name="startName_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Start Time:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="kubernetes_pod_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Kubernetes Pods:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="modelName_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLCDNumber" name="pods_lcdNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="Model_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Model:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="12" column="1" colspan="2">
        <widget class="QLCDNumber" name="fps_lcdNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="smallDecimalPoint">
          <bool>false</bool>
         </property>
         <property name="digitCount">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="2">
        <widget class="QLCDNumber" name="maxFPS_lcdNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="digitCount">
          <number>6</number>
         </property>
        </widget>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="maxFPS_label">
         <property name="styleSheet">
          <string notr="true">font: 75 36pt &quot;Ubuntu&quot;;
color: rgb(0, 0, 0);</string>
         </property>
         <property name="text">
          <string>MAX FPS:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="NGPU_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>GPUs/Pod:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="TotalGPU_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>GPUs:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="14" column="0">
        <widget class="QLabel" name="images_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
font: 75 36pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string>Img Processed:</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="15" column="1" colspan="2">
        <widget class="QLabel" name="StartTime_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLCDNumber" name="TotalGPU_lcdNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="digitCount">
          <number>3</number>
         </property>
        </widget>
       </item>
       <item row="14" column="1" colspan="2">
        <widget class="QLCDNumber" name="images_lcdNumber">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
         </property>
         <property name="digitCount">
          <number>15</number>
         </property>
         <property name="value" stdset="0">
          <double>0.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="16" column="1" colspan="2">
        <widget class="QLabel" name="elapsedTime_label">
         <property name="styleSheet">
          <string notr="true">color: rgb(170, 0, 0);
font: 75 28pt &quot;Ubuntu&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
